web/themes/custom/contribtracker/cypress.config.js

```diff
@@ -1,36 +1,9 @@
 import { defineConfig } from 'cypress';
-import { readFileSync } from 'fs';
-import path from 'path';
-// import cypressSplit from "cypress-split"; // Uncomment if you are using cypress-split
 
 export default defineConfig({
   projectId: 'contrib-tracker',
-  viewportWidth: 1512,
-  viewportHeight: 1200,
-  watchForFilesChanges: false,
-  chromeWebSecurity: false,
-  defaultCommandTimeout: 60000,
-  execTimeout: 10000,
-  pageLoadTimeout: 200000,
-  requestTimeout: 15000,
-  responseTimeout: 10000,
-  experimentalStudio: true,
-  trashAssetsBeforeRuns: false,
```

Comment:
@KosalaiV, why remove all this? This is not needed?

Reply:
> @KosalaiV, why remove all this? This is not needed?

@hussainweb Not needed in our case. As cypress by default handles timeouts. We can add few if needed.